<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;container for all the pages&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <widget class="QWidget" name="page_MainAbout">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QTextBrowser" name="textBrowser_MainAbout">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>620</width>
        <height>300</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'JetBrainsMono Nerd Font'; font-size:11pt; font-weight:500; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; text-decoration: underline;&quot;&gt;About&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;A collection of projects made in Qt 6. Made for learning C++.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; text-decoration: underline;&quot;&gt;Usage&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Actions&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;: Click &amp;quot;Actions&amp;quot; on the menu bar to open a drop down menu with a list of actions. Then you can press on the action to run it or use the keyboard shortcut.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Pages&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;: Click on a menu button in the menu bar, then in the drop down, click a page to go to it.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; text-decoration: underline;&quot;&gt;Projects&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Main&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;: Information about this entire project.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;MiniProjects&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;: Small single-page projects. Information about these projects are on their page.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;AESCryptography&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;: Allows you to encrypt and decrypt a secret plain text message using Advanced Encryption Standard. It uses SHA-256 to hash the passphrase and uses that as the key for AES. There are also hashing and conversion tools.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_MainCredits">
     <widget class="QTextBrowser" name="textBrowser_MainCredits">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>620</width>
        <height>300</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'JetBrainsMono Nerd Font'; font-size:11pt; font-weight:500; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; text-decoration: underline;&quot;&gt;Credits&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Made by: &lt;a href=&quot;https://github.com/AaronMisc/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#27bf73;&quot;&gt;AaronMisc&lt;/span&gt;&lt;/a&gt;.&lt;br /&gt;GitHub: &lt;a href=&quot;https://github.com/AaronMisc/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#27bf73;&quot;&gt;https://github.com/AaronMisc/&lt;/span&gt;&lt;/a&gt;.&lt;br /&gt;QT6ProjectCollection GitHub: &lt;a href=&quot;https://github.com/AaronMisc/QT6ProjectCollection&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#27bf73;&quot;&gt;https://github.com/AaronMisc/QT6ProjectCollection&lt;/span&gt;&lt;/a&gt;.&lt;br /&gt;Font: &lt;a href=&quot;https://www.jetbrains.com/lp/mono/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#27bf73;&quot;&gt;JetBrainsMono&lt;/span&gt;&lt;/a&gt; (&lt;a href=&quot;https://www.jetbrains.com/lp/mono/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#27bf73;&quot;&gt;https://www.jetbrains.com/lp/mono/&lt;/span&gt;&lt;/a&gt;).&lt;/p&gt;
&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; text-decoration: underline;&quot;&gt;Software&lt;/span&gt;&lt;/h3&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;IDE: Qt Creator.&lt;br /&gt;Framework: &lt;a href=&quot;https://www.qt.io/development/qt-framework/qt6&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#27bf73;&quot;&gt;Qt6&lt;/span&gt;&lt;/a&gt; (&lt;a href=&quot;https://www.qt.io/development/qt-framework/qt6&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#27bf73;&quot;&gt;https://www.qt.io/development/qt-framework/qt6&lt;/span&gt;&lt;/a&gt;).&lt;br /&gt;Language: C++.&lt;br /&gt;Build System: CMake.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMiniProjects">
    <property name="title">
     <string>MiniProjects</string>
    </property>
    <addaction name="actionMPNumberTracker"/>
   </widget>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionCredits"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAESCryptography">
    <property name="title">
     <string>AESCryptography</string>
    </property>
    <addaction name="actionAESCryptographyAbout"/>
    <addaction name="actionAESCryptographyConvert"/>
    <addaction name="actionAESCryptographyEncrypt"/>
    <addaction name="actionAESCryptographyHash"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuMain"/>
   <addaction name="menuMiniProjects"/>
   <addaction name="menuAESCryptography"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMPNumberTracker">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WindowNew"/>
   </property>
   <property name="text">
    <string>NumberTracker</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionCredits">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::AddressBookNew"/>
   </property>
   <property name="text">
    <string>Credits</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WindowClose"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAESCryptographyAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAESCryptographyConvert">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </action>
  <action name="actionAESCryptographyEncrypt">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemLockScreen"/>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </action>
  <action name="actionAESCryptographyHash">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::Scanner"/>
   </property>
   <property name="text">
    <string>Hash</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
